testGameSearch/GameService.swift

```diff
@@ -39,8 +41,10 @@ public class GameService {
                 if let d = data {
                     let gameLists = try JSONDecoder().decode([Game].self, from: d)
                       DispatchQueue.main.async {
-                        let game = gameLists
-                        //getting a yellow error here that says initialization of immutable value 'game' was never used.
+                        let game = gameLists.first
```

Comment:
It looks like you were defining game as a copy of gameLists but then the variable wasn't getting used by anything. Also, the completion handler here is looking for a single Game object, but gameLists is an array. For ease of testing here I have simply used first() but you may prefer to do something different.

Reply:
I was trying to get it to list all of the games that come with that name. So if someone searched Mario, it would show the first 5 in that array.